tests/sdk/test_git_url_utils.py

```diff
@@ -95,8 +95,7 @@ def test_with_password(self, monkeypatch: pytest.MonkeyPatch, git_url: GitURL):
 
         url = _git_url_utils.build_git_url(git_url)
         assert url == (
-            "https://git:<mocked secret>@github.com"
-            "/zapatacomputing/orquestra-workflow-sdk"
+            "https://git:<mocked secret>@github.com" "/zapata-engineering/orquestra-sdk"
```

Comment:
The split seems no longer necessary
```suggestion
            "https://git:<mocked secret>@github.com/zapata-engineering/orquestra-sdk"
```

Reply:
Thanks!